gatling/docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
You should add `set -e` after shebang line to exit script when first error occurs